wgrib2/ftn_api/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 
-
```

Comment:
Put some comments at the top of all CMakeLists.txt files. Mention that it is a CMake file from the wgrib2 project, and then specify that that CMake file builds. Add an author if you know it (Kyle Gerheiser for all CMake files that were already present in the build. Add your name to the list whenever you make significant changes or additions.